### PR TITLE
Refactor: tidy up error messages for all commands and decrease min size

### LINKED ISSUE #522: [PE-D] Incomplete screenshots
Screenshots do not show the endpoints portion of the listing. Could be helpful to show the added endpoint + its result in the command result display for `add`/`edit` functions

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: candyhy/ped#21

### LINKED ISSUE #584: Chore: standardise remove "Error:" from affected error messages
Some error messages have "Error:" in front of the message. Let's standardise to not have "Error:".

### LINKED ISSUE #585: Bug: error message differs for similar input between add/edit command
The error message for `edit 1 -x` and `add 1 -x` differs significantly. Let's standardise these to have a similar error message.

### PR DESCRIPTION
**Pull request by tjtanjin**
This PR does the following changes:
- Decrease min screen size to 800 x 600
- Tidy up and trim all error messages to not overlap with the GIFs but still contain minimal information
- Update user guide screenshots
- Fix paging in user guide

Fixes #522 
Closes #584 
Fixes #585

### CODE REVIEW

File: src/main/java/seedu/us/among/logic/parser/ParserUtil.java
Comment (tlylt) [inline src/main/java/seedu/us/among/logic/parser/ParserUtil.java:38]: Interesting that you passed the message into the `parseIndex` method to create a custom error message. I think the alternative is the previous structure which is fine as well, as the exception of an index parsing error need not know which command it is in when the exception occurs. It is when the error is being caught at the higher level, the exception is transformed with added details of the command usage. Let's keep to what you have since this change affects various test cases. Just some thoughts.

**Review by tlylt — COMMENTED** [review]
LGTM, @AY2021S2-CS2103T-T12-4/developers we can all do some manual testing to verify the error messages after merged, just to double confirm. I leave the final say to @JulietTeoh to approve the changes to the error messages.

File: src/main/java/seedu/us/among/logic/parser/ParserUtil.java
Comment (tjtanjin) [reply src/main/java/seedu/us/among/logic/parser/ParserUtil.java:38]: There... Was a previous method? xD Yea I agree what I did is actually not ideal, given more time (if we're serious about v2.0) we can clean this up in the future. On that note @JulietTeoh if you want to see how the error messages look like on the smallest window size (800 x 600) you may refer to the latest UG sent in our telegram group.

File: src/main/java/seedu/us/among/logic/commands/ShowCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/ShowCommand.java:26]: 'Use the help command for more information' can be refactored into MEsSSAGES. Would make changing this easier

File: src/main/java/seedu/us/among/logic/commands/ToggleCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/ToggleCommand.java:18]: same as above

File: src/main/java/seedu/us/among/logic/commands/SendCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/SendCommand.java:30]: same as below

File: src/main/java/seedu/us/among/logic/commands/RunCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/RunCommand.java:34]: same as below

File: src/main/java/seedu/us/among/logic/commands/RemoveCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/RemoveCommand.java:24]: same as below

File: src/main/java/seedu/us/among/logic/commands/AddCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/AddCommand.java:37]: same as below

File: src/main/java/seedu/us/among/logic/commands/EditCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/EditCommand.java:53]: same as below

File: src/main/java/seedu/us/among/logic/commands/FindCommand.java
Comment (ong6) [inline src/main/java/seedu/us/among/logic/commands/FindCommand.java:22]: same as below

**Review by ong6 — APPROVED** [review]
LGTM

**Pull request merged**
